packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -20,6 +20,8 @@ export const fetchFailureStatusCode: number = 710;
 export const invalidFileNameStatusCode: number = 711;
 // no response, or can't parse response
 export const fetchIncorrectResponse = 712;
+// Client and server epoch mismatch.
```

Comment:
I think we need to put good comment here explaining that this is not pseudo-code used only in client (like all 3 above), but it's a real code used by server (and client). This is not very obvious from reading code.

Reply:
Added elaborated description.